### PR TITLE
format the result of 'show resources'

### LINKED ISSUE #8932: Format result of "show resources"
## Feature Request
format result of `show resources;`

### Describe the feature you would like.
expect format of `show resources;` like this:
+------------+-------+-----------+------+-----------------------+-----------------+
| name      | type  | ip          | port | db                        |attribute         |
+------------+-------+-----------+------+-----------------------+-----------------+
| xxxx        | xxxxx | xxx       | xxx  | xxx                     |xxxx               |
+------------+-------+-----------+------+-----------------------+-----------------+

### PR DESCRIPTION
**Pull request by jingshanglu**
Fixes #8932 .

### CODE REVIEW

File: shardingsphere-proxy/shardingsphere-proxy-backend/src/main/java/org/apache/shardingsphere/proxy/backend/text/distsql/rql/impl/DataSourcesQueryBackendHandler.java [hunk 1]
```diff
@@ -42,6 +46,8 @@
     
     private Map<String, DataSourceParameter> dataSourceParameterMap;
     
+    private String schemaName;
```
Comment (terrymanu) [inline shardingsphere-proxy/shardingsphere-proxy-backend/src/main/java/org/apache/shardingsphere/proxy/backend/text/distsql/rql/impl/DataSourcesQueryBackendHandler.java:49]: It is not good a design for use global field which assigned out of the constructor.
Can you consider about improve it?

**Review by terrymanu — CHANGES_REQUESTED** [review]

File: shardingsphere-proxy/shardingsphere-proxy-backend/src/main/java/org/apache/shardingsphere/proxy/backend/text/distsql/rql/impl/DataSourcesQueryBackendHandler.java (hunk 1 above)
Comment (jingshanglu) [reply shardingsphere-proxy/shardingsphere-proxy-backend/src/main/java/org/apache/shardingsphere/proxy/backend/text/distsql/rql/impl/DataSourcesQueryBackendHandler.java:49]: ok.

**Review by tristaZero — CHANGES_REQUESTED** [review]
Is there any unit test?

**Review by terrymanu — APPROVED** [review]

**Comment by jingshanglu** [comment]
> Is there any unit test?

@tristaZero  Maybe Integration testing is required here，Unit testing cannot be effective，i have tested it manually.

**Review by tristaZero — APPROVED** [review]
This UT seems does not cover these key points and just gives a simple test no `ResponseHender`.  I have to say better than nothing.

**Pull request merged**